keel-sql/src/main/resources/jooqConfig.xml

```diff
@@ -20,6 +20,11 @@
           <name>LONGVARCHAR</name>
           <includeTypes>JSON</includeTypes>
         </forcedType>
+        <forcedType>
+          <userType>java.time.Instant</userType>
+          <includeTypes>TIMESTAMP</includeTypes>
+          <converter>com.netflix.spinnaker.jooq.LocalDateTimeToInstantConverter</converter>
+        </forcedType>
```

Comment:
Here's the bit of JOOQ configuration magic required

Reply:
LOVE